src/graph_compute/compute.cpp

```diff
@@ -15,7 +15,12 @@ using namespace std;
 #include "./common.h"
 #include "./concurrent_queue.h"
 
+WHEN_TEST(
+  volatile uint64_t roundUpdateCount;
```

Comment:
It was initialized just prior to calling execute_rounds, but I moved the initialization up to this point to avoid confusion.